qp_shogun/sortmerna/README.md

```diff
@@ -0,0 +1,134 @@
+# rRNA removal
+
+## SortMeRNA for MetaT analysis
+
+Updated  04/07/2020
+
+SortMeRNA for removal of ribsomal reads from quality filtered metaT data
+
+Latest SortMeRNA version: v2.1
+
+SortMeRNA info available at https://bioinfo.lifl.fr/RNA/sortmerna/
+
+Even if you are dealing with ribosomal RNA subtracted libraries, there will be some
+residual ribosomal RNA in the libraries that you want to remove/separate from
+the non ribosomal RNA sequences.
+
+Input: Quality filtered metaT reads (Fasta/Fastq) 
+
+## Building Custom databases
+
+
+Using the ARB package [2] to extract FASTA files for:
+- 16S bacteria, 16S archaea and 18S eukarya using SSURef_NR99_119_SILVA_14_07_14_opt.arb
+- 23S bacteria, 23S archaea and 28S eukarya using LSURef_119_SILVA_15_07_14_opt.arb.
+
+Remove partial sequences from all databases using HMMER 3.1b1 [4] (meta_RNA [5] HMM profiles used)
+
+
+Use SumaClust version 1.0.00 [6] to cluster sequences at various % id using the FASTA files generated in the previous step 
+
+
+Extract only the cluster centers
+
+  
+
+## Indexing custom databases
+
+```./indexdb_rna --ref db.fasta,db.idx ```
+
+Multiple databases can be separated by ":"
+
+
+## Currently indexed databases 
+
+rRNA databases are downloaded and stored at /opt/genome/qiita_dbs/sortmerna/rna-depletion/rRNA_databases. 
+
+Downloaded databases have been indexed and indexes are stored at /opt/genome/qiita_dbs/sortmerna/rna-depletion/index.
+
+rRNA from bacteria, archaea, eukarya (SILVA and rfam database)
```

Comment:
Good question! I think it will be great to have it in the main Qiita documentation only (which AFAIK) we already have as this will help avoid duplication and having users to search and find it within the plugins.

Reply:
Removed from the Plugin README version